firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigAutoFetch.java

```diff
@@ -195,7 +195,8 @@ public void run() {
 
   @VisibleForTesting
   public synchronized void fetchLatestConfig(int remainingAttempts, long targetVersion) {
-    Task<ConfigFetchHandler.FetchResponse> fetchTask = configFetchHandler.fetch(0L);
+    Task<ConfigFetchHandler.FetchResponse> fetchTask =
+        configFetchHandler.realtimeFetch(0L, FETCH_RETRY - remainingAttempts + 1);
```

Comment:
This could use a comment since the math is a bit cryptic on its own. Maybe even worth just keeping a separate attempts counter?

Slightly related, `FETCH_RETRIES_ALLOWED` or `INITIAL_FETCH_RETRIES` would be more descriptive as a variable name.

Reply:
Updated.